.gitpod.yml

```diff
@@ -0,0 +1,25 @@
+image:
+  file: .gitpod.Dockerfile
+
+tasks:
+  - init: |
+      go install -mod=readonly cuelang.org/go/cmd/cue
+      git clone https://github.com/cue-sh/vscode-cue /workspace/.vscode-remote/extensions/cuelang.org.cue.0.0.1
```

Comment:
Instead of this, personally I'm using these two from the marketplace:

https://marketplace.visualstudio.com/items?itemName=nickgo.cuelang
https://marketplace.visualstudio.com/items?itemName=jallen7usa.vscode-cue-fmt

Reply:
Gitpod can't use extensions from the Microsoft marketplace, due to licensing restrictions https://www.gitpod.io/docs/ides-and-editors/vscode-extensions#where-do-i-find-extensions. The two extensions you mentioned are also what I use locally but they are not listed on Open VSX yet. I've added issues to each repo to do so, but that could take some time.

Reply:
I'll comment the code to make that clear.